lib/ex_unit/lib/ex_unit/server.ex

```diff
@@ -91,7 +108,15 @@ defmodule ExUnit.Server do
       when is_integer(loaded) do
     state =
       if uniq? do
-        async_modules = :queue.to_list(state.async_modules) |> Enum.uniq() |> :queue.from_list()
+        async_modules =
+          state.async_modules
+          |> :queue.to_list()
+          |> Enum.uniq()
+          |> Enum.map(fn {group, modules} ->
+            {group, Enum.uniq(modules)}
+          end)
```

Comment:
I think this line is wrong now, because we don't have the groups at this moment. We would need to traverse all groups in the map and make them unique too.

```suggestion
```

Reply:
I've pushed https://github.com/elixir-lang/elixir/pull/13897/commits/fe3f66c5bfe18e07d1434985f07e0997431d987b to make it work with the new state format.